common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON

```diff
@@ -0,0 +1,1256 @@
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json"
+								  },
+								  {
+									  "key": "X-chkp-sid",
+									  "value": "{{session}}"
+								  }
+							  ],
+							  "body": {
+								  "mode": "raw",
+								  "raw": "{\"subscription\": \"<subscription_id>\", \"application_id\": \"<application_id>\", \"directory_id\": \"<directory_id>\", \"client_secret\": \"<client_secret>\", \"deletion_tolerance\": \"<deletion_tolerance>\"}",
+								  "options": {
+									  "raw": {
+										  "language": "json"
+									  }
+								  }
+							  },
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/accounts/azure/<account_name>",
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "accounts",
+									  "azure",
+									  "<account_name>"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "POST add Azure gw configuration",
+						  "request": {
+							  "method": "POST",
+							  "header": [
+								  {
+									  "key": "X-chkp-sid",
+									  "value": "{{session}}",
+									  "type": "text"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "body": {
+								  "mode": "raw",
+								  "raw": "{\"name\": \"<gwConfigurations_name>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+							  },
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/azure",
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations",
+									  "azure"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "PUT set Azure gw configuration",
+						  "request": {
+							  "method": "PUT",
+							  "header": [
+								  {
+									  "key": "X-chkp-sid",
+									  "value": "{{session}}",
+									  "type": "text"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "body": {
+								  "mode": "raw",
+								  "raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+							  },
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/azure/<gwConfigurations_name>",

```

Comment:
Please change to "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/azure/<**gwConfiguration_name**>",